src/main/java/seedu/address/logic/commands/task/TeditCommand.java

```diff
@@ -31,7 +31,7 @@
     public static final String COMMAND_WORD = "tedit";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the detail of the task identified"
-            + "by the corresponding index number.\n"
+            + " by the corresponding index number.\n"
```

Comment:
Maybe try adding the spacebar at the end in line 33 instead. 

e.g. ": Edits the detail of the task identified "